apps/avatax/src/pages/api/manifest.ts

```diff
@@ -32,7 +32,7 @@ export default wrapWithLoggerContext(
           homepageUrl: "https://github.com/saleor/apps",
           id: "saleor.app.avatax",
           name: "Avatax",
-          permissions: ["HANDLE_TAXES", "MANAGE_ORDERS"],
+          permissions: ["HANDLE_TAXES", "MANAGE_ORDERS", "MANAGE_DISCOUNTS"],
```

Comment:
question: will the migration that adds this permission via appbridge be added in a separate PR or are we just going to have to re-instal the app?

Reply:
I hope we will fix this on API level